src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Alerting concepts and terms
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Incident intelligence
+metaDescription: 'Some basic terms and concepts for New Relic alerting and applied intelligence.'
+---
+
+import alertConceptsDiagram from 'images/alerting-concepts-diagram.png'
+
+To make the best use of New Relic alerting and applied intelligence features, it will help you to understand some basic terms and concepts. 
+
+<img
+  title="A diagram showing some basic concepts and terms for New Relic alerting and applied intelligence"
+  alt="A diagram showing some basic concepts and terms for New Relic alerting and applied intelligence"
+  src={alertConceptsDiagram}
+/>
+<figcaption>A diagram showing how violations of alert **conditions** result in **incidents**, how incidents result in **issues**, and how the logic in **workflows** results in notifications.   
+  </figcaption>
+
+You set up alerts for your telemetry data by configuring alert **conditions**, which define what will constitute a violation and trigger an **incident**. For example, you might want to create an alert condition that opens an incident if a host's CPU goes above 80%. Incidents result in an **issue**, which is the thing that actually results in notifications.
+
+Conditions live inside an alert **policy**. The policy contains higher-level settings that govern all the conditions in that policy. One setting is **issue creation preference**, which governs how multiple incidents can get grouped into a single issue. For example, you might decide that every incident gets its own issue, or you might group them by condition, or you might group them by policy. We have built-in **decisions**, which are algorithms that automatically group incidents into issues, and you can create your own custom decisions. 
```

Comment:
Decisions automatically merge issues. The only thing that groups incidents into issues is the issue creation preference on a policy.